pyuvdata/uvdata/mir_parser.py

```diff
@@ -3727,9 +3870,9 @@ def read_vis_data(self, return_vis=True, use_mmap=True, read_only=False):
         # Figure out which results we need to pass back
         return data_dict
 
```

Comment:
Made it to here! I will pick it back up tomorrow.

Reply:
Thanks!